web/src/components/cards/BuildingCard.vue

```diff
@@ -139,8 +139,8 @@ tryOrAlertAsync(async () => {
   can_expand.value = can_expand.value && progresses.value.length > 0;
 });
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function route(date: Date | null) {
-  console.log(`TODO: link to ${date}`);
```

Comment:
Dit gewoon weglaten is gevaarlijk. Ik zou tenminste de TODO in comment zetten of een issue aanmaken.